.github/workflows/e2e.yml

```diff
@@ -0,0 +1,130 @@
+      run: |
+        cd ../
+        git clone https://github.com/dell/csi-baremetal-operator.git 
+
+    - name: Set environment variables
+      run: |
+        echo "CSI_BAREMETAL_DIR=$RUNNER_WORKSPACE/csi-baremetal/" >> $GITHUB_ENV
+        echo "CSI_BAREMETAL_OPERATOR_DIR=$RUNNER_WORKSPACE/csi-baremetal-operator/" >> $GITHUB_ENV
+
+    - name: LVM2 install
+      run: sudo apt-get install -y lvm2
+
+    - name: Prepare Golang 
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{env.go_version}}
+
+    - name: Install helm
+      run: |
+        curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
+        sudo apt-get install apt-transport-https --yes
+        echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+        sudo apt-get update
+        sudo apt-get install helm
+
+    - name: CUSTOM KIND
+      run: |
+        wget -O kind https://github.com/painhardcore/builder/releases/download/0.0.1-alpha/kind_v0.8.1
+        sudo chmod +x kind
+        sudo mv kind /usr/bin
+          
+    - name: Kubectl install
+      run: |
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+        chmod +x ./kubectl
+        sudo mv ./kubectl /usr/local/bin/kubectl
+        kubectl version --client 
+
+    - name: add CSI_VERSION
+      run: |
+        cd ${{env.CSI_BAREMETAL_DIR}}
+        echo "CSI_VERSION=`make version`" >> $GITHUB_ENV
+
+    - name: Get dependencies
+      run: make dependency
+
+    - name: Compile proto files
+      run: |
+        sudo apt install protobuf-compiler
+        sudo apt install golang-goprotobuf-dev
+        make install-compile-proto
+
+    - name: Generate CRD
+      run: |
+        make install-controller-gen 
+        make generate-deepcopy
+
+    - name: Build binary
+      run: |
+        make build
+        make DRIVE_MANAGER_TYPE=loopbackmgr build-drivemgr
+
+    - name: Download health-probe
+      run: make download-grpc-health-probe
+
+    - name: Build docker images
+      run: |
+        make images REGISTRY=${{ env.REGISTRY }}
+        make DRIVE_MANAGER_TYPE=loopbackmgr image-drivemgr REGISTRY=${{ env.REGISTRY }}
+    
+    - name: docker-build operators 
+      run: |
+        cd ${{env.CSI_BAREMETAL_OPERATOR_DIR}}
+        echo "CSI_OPERATOR_VERSION=`make version`" >> $GITHUB_ENV
+        make docker-build REGISTRY=${{ env.REGISTRY }}
+
+    - name: Kind preparation
+      run: |
+        cd ${{env.CSI_BAREMETAL_DIR}}
+        make kind-build
+        make kind-create-cluster
+        kubectl cluster-info --context kind-kind
+        kubectl get pods -o wide --all-namespaces 
+        echo "current-context:" $(kubectl config current-context)
+        echo "environment-kubeconfig:" ${KUBECONFIG}
+        sudo sysctl net/netfilter/nf_conntrack_max=131072 
```

Comment:
Could you try kind 0.11.1 and the latest kind-node for k8s 1.19 [(link to the release)](https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1) and remove system env editing?
For installing kind - `make kind-build KIND_VER=0.11.1`
Kind-node version is passed [here](https://github.com/dell/csi-baremetal/blob/2a1958a294373eaf8450d2598abe26b1de6a33be/Makefile.validation#L141). Please create Makefile environment for it

Reply:
Added